de.peeeq.wurstscript/src/test/java/tests/wurstscript/tests/ClassesTests.java

```diff
@@ -1114,5 +1114,24 @@ public void tupleArrayMember() {
         );
     }
 
+    @Test
+    public void testOver9000() {
+        testAssertOkLines(true,
+                "package Vegeta",
+                "   native testSuccess()",
+                "   class Scouter",
+                "       static int size = 0",
+                "       construct()",
+                "           size++",
+                "",
+                "   init",
+                "       for i = 0 to 20000",
+                "           new Scouter()",
+                "       if Scouter.size == 20001",
+                "	        testSuccess()",
```

Comment:
Wouldn't it make more sense to create 20k powelevel objects instead of 20 scouters?